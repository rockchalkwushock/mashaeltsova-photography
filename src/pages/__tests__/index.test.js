import React from 'react'
import { mount, shallow } from 'enzyme'

import HomePage from '../'

describe('Pages: HomePage', () => {
  test('should render markup', () => {
    const wrapper = mount(<HomePage />)
    expect(wrapper).toMatchSnapshot()
  })
  test('should have props"', () => {
    const wrapper = shallow(<HomePage />)
    expect(wrapper.find('Title').props()).toHaveProperty(
      'text',
      'Refactor Engage!'
    )
  })
})
