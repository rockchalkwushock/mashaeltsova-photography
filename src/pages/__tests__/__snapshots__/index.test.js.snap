// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages: HomePage should render markup 1`] = `
.c0 {
  color: lightcoral;
}

<IndexPage>
  <div>
    <Title
      text="Refactor Engage!"
    >
      <StyledTitle
        color="lightcoral"
      >
        <h1
          className="c0"
          color="lightcoral"
        >
          Refactor Engage!
        </h1>
      </StyledTitle>
    </Title>
  </div>
</IndexPage>
`;
